decidim-core/spec/system/search_spec.rb

```diff
@@ -32,6 +32,14 @@
       expect(page).to have_content(/results for the search: "#{term}"/i)
       expect(page).to have_selector(".filter-search.filter-container")
     end
+
+    it "has all the resources to search" do
+      within ".search__filter" do
+        expect(page).to have_content("All")
+        expect(page).to have_content("Participants")
+        # TODO: all the resources
```

Comment:
What I meant here is to add the other resources. Something like:

```ruby
expect(page).to have_content("Groups")
expect(page).to have_content("Participatory processes")
```

But also all the other Resources. The idea is that if the "Projects" line wouldn't be here, this spec would have failed.

Reply:
So to test for everything other than "Projects", that appear in a search?

Reply:
To test everything that's currently in the view with this PR.

So, it'd be the list of the "Before" screenshot with also the "Projects" (respecting the order).

The intention is to detect if this change is rolled back for some reason, like a bad merge resolution in `decidim-core/lib/decidim/searchable.rb`.

Reply:
Gotcha